torch/csrc/distributed/rpc/init.cpp

```diff
@@ -356,32 +356,36 @@ If the future completes with an error, an exception is thrown.
   module.def(
       "_invoke_rpc_torchscript",
       [](const std::string& dstWorkerName,
-         const std::string& qualifiedNameStr,
-         const py::args& args,
-         const py::kwargs& kwargs) {
-        DCHECK(!PyGILState_Check());
```

Comment:
@wanchaol 

Because we used to call

```pythoon
_invoke_rpc_torchscript(
    dst_worker_info.name, torch._jit_internal._qualified_name(func), *args, **kwargs
)
``` 

Now in this PR, I changed it to 

```python
_invoke_rpc_torchscript(
    dst_worker_info.name, func, args, kwargs
)
```